source/extensions/omni.isaac.lab_tasks/config/extension.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 # Note: Semantic Versioning is used: https://semver.org/
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
Could you also update the `CHANGELOG.rst` file?

Reply:
Changed it :D